package.json

```diff
@@ -52,6 +52,7 @@
     "aws-sdk": "^2.1379.0",
     "commander": "^11.0.0",
     "dotenv": "^16.0.3",
-    "eslint-plugin-vitest": "^0.3.9"
+    "eslint-plugin-vitest": "^0.3.9",
+    "otplib": "^12.0.1"
```

Comment:
these 2 are dev deps, not deps. Same for `@vitest/coverage-v8`, `dotenv` and `amazon-cognito-identity-js` (if the latter is only used for types)